docs/pages/components/breadcrumb.md

```diff
@@ -41,21 +41,24 @@ there should be popover with missing options.
                     <span class="fd-breadcrumb__dropdown-icon"></span>
                 </a>
             </div>
-            <div class="fd-popover__body fd-popover__body--no-arrow" aria-hidden="true" id="breadcrumb1">
-                <ul class="fd-list fd-list--compact">
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
-                    </li>
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>                    
-                    </li>
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
-                    </li>
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
-                    </li>
-                </ul>
+            <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--breadcrumb" aria-hidden="true" id="breadcrumb1">
```

Comment:
In my opinion this is not the best approach. Inside popover you shouldn't have classes that are specific for other components like `fd-popover__body--breadcrumb`. Isn't it possible to do it like this:
```
<div class="fd-popover__body fd-popover__body--no-arrow fd-breadcrumb__popover-body" >
```

In this case `fd-breadcrumb__popover-body` will bring the rounded corners and the code will be in the breadcrumb.scss file. Also I'd suggest testing this in NGX since it also involves popper.js and thing not always work out of the box there.

Reply:
I had thought about that, but i did some future preventative measures thinking that in the future it would need to use this attribute. I've noticed they use either 0.25 or 0.125. Maybe name it fd-popover__body--alt-radius? @InnaAtanasova

Reply:
`fd-popover__body--breadcrumb` should not be used inside other components. This is breadcrumb specific property (even if other components have the same value for rounded corners).

Reply:
Alright will fix